_shows/21_22/freedom_from.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Freedom From
+playwright: Marianna Whistlecroft
+student_written: true
+period: Spring
+season: Fringe
+season_sort: 230
+date_start: 2022-05-01
+date_end: 2022-05-03
+venue: New Theatre Studio A
+
+cast:
+- role: Daisy
+  name: Hettie Rockell
+- role: Van
+  name: Kieran Hatchett
+- role: Louis and Games Master
+  name: Vicent Solbes
+
+crew:
+- role: Director
+  name: Marianna Whistlecroft
+- role: Theatre Director
+  name: Beth White
+- role: Artistic Director
```

Comment:
What is the difference between a theatre director and an artistic director for a play? 
The difference may be worth putting in comments or notes, or setting:
```
- role: Director 
  name: Beth White
  note: Theatre
- role: Director
  name: Sharon Anatole
  note: Artistic
```

Reply:
not completely sure of the difference - it was what they had written in their programme